es7/2016-07/jul-26.md

```diff
@@ -24,7 +24,7 @@ RW: If we have different note-takers, they might not follow the right format, ma
 
 DE: Let's defer to Brian when he gets back
 
-IS: Different Ecma groups do different things; there is no single solution. Some e.g. only record the subject and outcome of the discussion, but not the actual discussion itself. This has to be decided by the TC. Anyway, with the current practice, which is very similar what the W3C does, we need the support of the group. e.g. Ecma can not hire any pay for a professional technical notetaker, that has to be organized / fulfilled by TC39 members.
+IS: Different Ecma groups do different things; there is no single solution. Some e.g. only record the subject and outcome of the discussion, but not the actual discussion itself. This has to be decided by the TC. Anyway, with the current practice, which is very similar what the W3C does, we need the support of the group. e.g. ECMA can not hire any pay for a professional technical notetaker, that has to be organized / fulfilled by TC39 members.
```

Comment:
There's other "Ecma"s in here - but actually, that's the correct capitalization (see http://www.ecma-international.org)